src/views/templates/utils/constants.ts

```diff
@@ -35,3 +35,14 @@ export const NO_EDIT_TEMPLATE_PERMISSIONS = 'Can not edit in view-only mode';
 
 // t('Can not delete in view-only mode')
 export const NO_DELETE_TEMPLATE_PERMISSIONS = 'Can not delete in view-only mode';
+
+import { VirtualMachineDetailsTab } from '@kubevirt-utils/constants/tabs-constants';
+
+export const templateTabsWithYAML = [
+  VirtualMachineDetailsTab.Configurations,
+  VirtualMachineDetailsTab.Disks,
+  VirtualMachineDetailsTab.NetworkInterfaces,
+  VirtualMachineDetailsTab.Scheduling,
+  VirtualMachineDetailsTab.Scripts,
+  'parameters',
```

Comment:
Did you see `'parameters'` under `VirtualMachineDetailsTab`? I didn't...

Reply:
What I meant is that only parameters are hardcoded in this array. Please check if it can be replaced by an already declared const so if something is changed, we wont need to change in a few places